spec/support/dependency_helpers.rb

```diff
@@ -52,11 +52,12 @@ def build_git_gem(gem_name, version = "1.0.0")
     build_gem gem_name, version
 
     Dir.chdir "#{TMP_GEM_BUILD}/#{gem_name}" do
-      `git init . --initial-branch=master`
-      `git config user.email "appraisal@thoughtbot.com"`
-      `git config user.name "Appraisal"`
-      `git add .`
-      `git commit --all --no-verify --message "initial commit"`
+      %x[git init . --initial-branch=master]
+      %x[git config user.email "appraisal@thoughtbot.com"]
+      %x[git config user.name "Appraisal"]
+      %x[git config commit.gpgsign false]
```

Comment:
On my local machine I have git configured to sign all commits, and this was breaking the test suite here, because it couldn't sign the commit two lines below.  Adding this config to prevent signing for this ephemeral gem fixed the issue.